zeus/carbon/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""Carbon intensity providers used for carbon-aware optimizers."""
+
+from zeus.carbon.electricity_maps_carbon_intensity_provider import (
+    ElectricityMapsCarbonIntensityProvider,
+)
+
+import requests
+
+
+def get_ip_lat_long() -> tuple[float, float]:
+    """Retrieve the latitude and longitude of the current IP position."""
+    try:
+        ip_url = "http://ipinfo.io/json"
+        resp = requests.get(ip_url)
+        loc = resp.json()["loc"]
+        lat, long = map(float, loc.split(","))
+        print(f"Retrieve latitude and longitude: {lat}, {long}")
```

Comment:
Libraries should rarely use `print` directly. Please refer other parts of the library to learn how `logging`/`logger` is used.